charts/kyverno/values.yaml

```diff
@@ -51,9 +51,19 @@ priorityClassName: ""
 antiAffinity:
   # This can be disabled in a 1 node cluster.
   enable: true
-  # By default this will make sure two pods don't end up on the same node
-  # Changing this to a region would allow you to spread pods across regions
```

Comment:
should we keep the comment?

Reply:
If so, where do you want to place the comment? Over the typologyKey again?